docs/development.md

```diff
@@ -114,9 +114,12 @@ If it is not a valid version try, make sure you have updated the godep repo with
 
 ```console
 cd $GOPATH/src/github.com/skippbox
-godep restore
+script/godep-restore.sh
 ```
 
+Reason for calling `script/godep-restore.sh` instead of just `godep restore` is that Kompose is using `github.com/openshift/kubernetes` instead of `github.com/kubernetes/kubernetes`.
```

Comment:
@kadel Is there any way to just using `godep restore` ? currently we are using openshift/kubernetes to support OpenShift provider, what if we upgrade kubernetes to next releases 1.4, 1.5, etc ?